src/app-layout/visual-refresh/context.tsx

```diff
@@ -545,6 +564,8 @@ export const AppLayoutInternalsProvider = React.forwardRef(
           toolsRefs,
           navigationRefs,
           loseToolsFocus,
+          isOverlapDisabled,
+          hasBreadcrumbs: !!props.breadcrumbs,
```

Comment:
I don't think we need this. You can just use `breadcrumbs` directly which is already in the the context.